src/app/pages/clients/clients.component.html

```diff
@@ -0,0 +1,61 @@
+<div>
+  <div class="row">
+    <div class="col-md-8">
+      <div class="row">
+        <div class="col-md-6" *ngFor="let client of clients">
+          <div class="card listview">
+            <div class="row no-gutters">
+              <div class="col">
+                <div class="card-block px-2">
+                  <h4>
+                    <b class="card-title"> {{ client.client_name }}</b>
+                  </h4>
+
+                  <ul class="list-unstyled fa-ul">
+                    <li>
+                      <i class="fa fa-street-view"></i>
+                      <span
+                        >{{ client.address.Street }}, {{ client.address.City }},
+                        {{ client.address.State }}
+                      </span>
+                    </li>
+                    <li>
+                      <i class="fa fa-phone"></i>
+                      <span>{{ client.phone }}</span>
+                    </li>
+                    <li>
+                      <i class="fa fa-envelope"></i
+                      ><span>{{ client.email }}</span>
+                    </li>
+                  </ul>
+                </div>
+              </div>
+            </div>
+            <div class="card-text">
+              <p>
+                <b class="client-reviews">Reviews</b>
+
+                <span class="row">
+                  <a href="/client/:clientId/review" class="card-link col"
```

Comment:
I would suggest to change 
``href="/client/:clientId/review"``
to
``[routerLink]="['/client',client.id,'review']"``.
So that we will have a dynamic route depending on the client.

Reply:
Thanks, @Niyitangasam  I would need to add it the app routing to do that which goes beyond what my code is intended to deliver.

Reply:
@Niyitangasam it's better those links are left static so that the developer who is to use them to navigate to a feature they are working can make them dynamic as it is also part of his task to test the navigation from @Rhytah feature to theirs... so @Rhytah leave them static for now